mix.exs

```diff
@@ -36,7 +37,7 @@ defmodule Rclex.MixProject do
       extra_applications:
         [:logger] ++
           case Mix.target() do
-            :host -> [:runtime_tools, :wx, :observer]
```

Comment:
We use `:observer` which needs alse `:wx` for development, please do not remove them

Reply:
Thank you for your attention. I removed it, because wx was on available in my container environment and forgot to readd. Having the observer available during development/testing is really useful.